src/main/java/kg/jarkyn/cobspecserver/responders/OptionsResponder.java

```diff
@@ -0,0 +1,20 @@
+package kg.jarkyn.cobspecserver.responders;
+
+import kg.jarkyn.cobspecserver.data.Request;
+import kg.jarkyn.cobspecserver.data.Response;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class OptionsResponder extends Responder {
+    private List<String> methods = Arrays.asList("GET", "HEAD", "POST", "OPTIONS", "PUT");
```

Comment:
Was this implementation to make a cob spec test pass or does your server support these methods?